src/main/scala/codacy/detekt/YamlConfigFactory.scala

```diff
@@ -0,0 +1,12 @@
+package codacy.detekt
+
+import io.gitlab.arturbosch.detekt.api.internal.YamlConfig
+
+object YamlConfigFactory {
+
+  def create(properties: java.util.Map[String, _]): YamlConfig = {
+    val constructor = classOf[YamlConfig].getConstructors.head
```

Comment:
Codacy found an issue: [Usage of head on collections.](https://app.codacy.com/gh/codacy/codacy-detekt/pullRequest?prid=5879941)